api/src/wfl/module/copyfile.clj

```diff
@@ -101,7 +101,7 @@
 (defn create-copyfile-workload!
   "Use transaction TX to add the workload described by REQUEST."
   [tx {:keys [items common] :as request}]
-  (letfn [(nil-if-empty [x] (if (empty? x) nil x))
+  (letfn [(nil-if-empty [x] (when (seq x) x))
```

Comment:
that's just `not-empty`

Reply:
Clojure says that `(seq x)` is the preferred idiom for testing a non-empty collection: https://clojuredocs.org/clojure.core/seq#example-542692d5c026201cdc327079

Reply:
Good to know.

Reply:
I like `not-empty` for this.

Reply:
FYI: `seq` returns a sequence on the collection, but `not-empty` returns the original collection.